kurtosis-devnet/cmd/main.go

```diff
@@ -68,6 +73,38 @@ func writeEnvironment(path string, env *kurtosis.KurtosisEnvironment) error {
 	return nil
 }
 
+func writeConductorConfig(path string, enclaveName string) error {
+	if path == "" {
+		return nil
```

Comment:
Shouldn't we return an error in this case?
And perhaps also the case where `enclaveName == ""`?

I see you're using it like so `writeConductorConfig(cfg.conductorConfig, cfg.enclave)` which suggests you want the default to be a no-op? If so should we document this more clearly? I found this a bit surprising; wouldn't we always want a conductor config to be written?

Reply:
I just found it a bit unclear. But no file when no conductors sounds fine. Same as empty file. As long as we're clear, so people know what to expect.
Just my opinion of course :)